iOS_SDK/OneSignalDevApp/OneSignalDevApp/OneSignalExample-Bridging-Header.h

```diff
@@ -0,0 +1,13 @@
+//
+//  OneSignalExample-Bridging-Header.h
+//  OneSignalExample
+//
+//  Created by Elliot Mawby on 12/12/22.
+//  Copyright © 2022 OneSignal. All rights reserved.
```

Comment:
Fix copyright header to match our standard.

Reply:
done!